### ISSUE TITLE
Fix postcss vulnerability

### ISSUE THREAD

**Issue by felipeqq2** [csstools/postcss-preset-env#207]
The `postcss` package has a security vulnerability on v7 (CVE-2021-23368). It would be interesting if the package is updated to its latest version. 
```
postcss  7.0.0 - 8.2.9
Severity: moderate
Regular Expression Denial of Service - https://npmjs.com/advisories/1693
```
I think the [breaking changes](https://github.com/postcss/postcss/releases/tag/8.0.0) for v8 will not cause any problems.

**Comment by kevinlandsberg** [comment]
![image](https://user-images.githubusercontent.com/46755225/119844473-bdce7580-bf08-11eb-883c-b8627056f626.png)

**Comment by felipeqq2** [comment]
PostCSS released a patch for v7 - 7.0.36
